pkgs/development/compilers/ponyc/default.nix

```diff
@@ -0,0 +1,46 @@
+{stdenv, glibc, fetchgit, llvm, makeWrapper, openssl, pcre2 }:
+
+stdenv.mkDerivation {
+  name = "ponyc-0.2.1";
+
+  src = fetchgit {
+    url = "https://github.com/CausalityLtd/ponyc";
+    rev = "refs/tags/0.2.1";
```

Comment:
Could you use fetchFromGithub instead, or fetchurl for the tarball, AFAIK it saves some space on hydra.